src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -80,6 +80,26 @@ export interface ChartContainerProps<T extends ChartDataTypes> {
   plotContainerRef: React.RefObject<HTMLDivElement>;
 }
 
+interface XAxisProps {
```

Comment:
minor (can be done with the next PR): These could inherit from a shared interface that contains axis, tickCount, tickFormatter, title, ariaRoleDescription.